### PR TITLE
Add my HTML into the main branch

### PR DESCRIPTION
**Pull request by ShaveYourEyebrows**
PLEEEEEEEEEEEEEEEEEEEEEEEEEEEEASE

OTHERWISE WE'LL HAVE TO REBASE EVERYTHINGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG

### CODE REVIEW

**Review by carthy27 — APPROVED** [review]
Love the new pic. Lgtm

File: OBlock/contracts/StudentContract.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract StudentContact{
+
+    mapping(string => Student) studentMap;
+
+    struct Student{
+        uint256 id;
+        uint8 qca;
```
Comment (Bourkupine) [inline OBlock/contracts/StudentContract.sol]: just thinking here, we shouldn't have to add a students qca? that should be handled elsewhere (and potentially outside the scope for the project)

File: OBlock/contracts/StudentContract.sol [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract StudentContact{
+
+    mapping(string => Student) studentMap;
+
+    struct Student{
+        uint256 id;
```
Comment (Bourkupine) [inline OBlock/contracts/StudentContract.sol]: surely uint256 is far too big

**Review by Bourkupine — COMMENTED** [review]
Left some comments

File: OBlock/contracts/StudentContract.sol (hunk 1 above)
Comment (oOoOoOo0o) [reply OBlock/contracts/StudentContract.sol]: Agree

Comment (ShaveYourEyebrows) [reply OBlock/contracts/StudentContract.sol]: Fixed!!!!

File: OBlock/contracts/StudentContract.sol (hunk 2 above)
Comment (ShaveYourEyebrows) [reply OBlock/contracts/StudentContract.sol]: Changed to a uint32

**Review by oOoOoOo0o — APPROVED** [review]

**Review by Bourkupine — APPROVED** [review]
Looks good, happy to approve.

**Pull request merged**